src/api/create-vis.js

```diff
@@ -64,13 +64,19 @@ var loadVizJSON = function (el, visModel, vizjsonData, options) {
     showLayerSelector = vizjson.options.layer_selector;
   }
 
+  var isEmbed = true;
```

Comment:
IMO, we shouldn't use the term `embed` in the context of CartoDB.js.... I'd try to find a more generic name...